api/src/test/java/org/openmrs/module/initializer/api/DispositionsLoaderIntegrationTest.java

```
@@ -0,0 +1,28 @@
+package org.openmrs.module.initializer.api;
+
+import org.junit.Test;
+import org.openmrs.module.emrapi.disposition.DispositionService;
+import org.openmrs.module.initializer.DomainBaseModuleContextSensitiveTest;
+import org.openmrs.module.initializer.api.loaders.DispositionsLoader;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DispositionsLoaderIntegrationTest extends DomainBaseModuleContextSensitiveTest {
+	
+	@Autowired
+	protected InitializerService iniz;
+	
+	@Autowired
+	private DispositionsLoader loader;
+	
+	@Autowired
+	private DispositionService dispositionService;
+	
+	@Test
+	public void load_shouldLoadDisposition() {
+		loader.load();
+		assertEquals(5, dispositionService.getDispositions().size());
+	}
```

Seems like we should have 1 or 2 other tests.  eg:
* Test that dispositions are always reloaded every time
* Test that an exception is thrown if multiple files are found in the domain

---

New tests added, see above.

It's slightly hacky that the duplicate config file is present for all three of the tests above so we can test the "exception" case, and we are relying on the fact that Iniz ignores exceptions by default test the happy paths.

---

I also confirmed that the reload test does fail if we don't have checksums properly disabled.

---

Oh,nice, thanks @Ruhanga , I can try that.